source/util.js

```diff
@@ -68,3 +68,27 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.getPreId = pMemoize(async options => {
+	ow(options, ow.object.hasKeys('yarn'));
```

Comment:
Doesn't this make the `npm` code below moot as non-yarn wouldn't pass?

Reply:
I think not. Since we are setting `options.yarn` using your `has-yarn`. package earlier, all we are checking is if the `options` object has `yarn` key using `ow.object.hasKeys` so that it doesn't throw an error later.